### ISSUE TITLE
there are no supply factories on south island sahrani

### ISSUE THREAD

**Issue by dvdbrewster** [Wyqer/kp_liberation#281]
### Basic Information
**Mission version:**961
**Map used:**sara
**Mods used:**projOpfor,ace.cba
**Dedicated or local game?**ded
**GitHub or Workshop mission file?**git

### Individual Things
**Did you edit anything within the mission files?**
not yet

**If yes, what did you edit exactly?**
?

### Short Issue Description
Theres not a single supply (-f)  on south sahrani.   I wish to add a couple(change)

### Expected Behaviour
some
just tell me where I can make changes?
### Steps to Reproduce the Issue
all

**Comment by Wyqer** [comment]
There are 6 factories on the south island of sahrani. I see no need to add some / transform other sectors to factory sectors.

**Comment by veteran29** [comment]
If by saying "supply (-f)" u meant there is no single factory that produces _supply resource_, this will vary on every save. What is produced in factories is randomed every time new game is started.

As @Wyqer said there is no need for more factories on south sahrani.

**Comment by dvdbrewster** [comment]
OK. I suppose you  careless if I agree or am satisfied. This may be time consuming, so Ill just be complete and accept your resolve. Its your creation. I've restarted with  save-wipe a couple times. I've found my own FOB location.. this is best if resources and manipulation are a priority. I/ve not witnessed the supply types changing on  reboot. I/ve usually (on s, Island) had to move a supply crate from base to one of the AMMO FACTORIES  and create a supply factory. I profess this is normal for that side of SARA.  I just say, that more than once  theres been no default -s  factory and itsa a pain in the butt.
